### PR TITLE
Add a description of '--internal' to the docs

### PR DESCRIPTION
**Pull request by robmry**
**- What I did**

Add a description of a --internal network (written by @neersighted) to the CLI docs - as they currently only mention overlay networks.

(Also, while I was there, update the description for the table listing 'network create' options, to make it consistent with the table listing 'bridge' options.)

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Added a description of --internal networks to the docs.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/commandline/network_create.md [hunk 1]
```diff
@@ -183,6 +184,13 @@ $ docker network create \
 
 ### <a name="internal"></a> Network internal mode (--internal)
 
+Containers on an internal network may communicate between each other, but are
+precluded from communicating with any networks the host has access to (LAN or
+WAN) as no default route is configured, and firewall rules are set up to drop
+all outgoing traffic. Communication with the gateway IP address (and thus
+appropriately configured host services) is possible, and the host may
+communicate with any container IP directly.
```
Comment (dvdksn) [inline docs/reference/commandline/network_create.md:192]: Can we make the first sentence shorter and easier to read for non-native readers? For example:

```suggestion
Containers on an internal network may communicate between each other,
but not on any other network, as no default route is configured,
and firewall rules are set up to drop all outgoing traffic.
Communication with the gateway IP address (and thus
appropriately configured host services) is possible, and the host may
communicate with any container IP directly.
```

Comment (robmry) [reply docs/reference/commandline/network_create.md]: Thanks David ... I modified that slightly, hopefully it's ok?

**Review by dvdksn — APPROVED** [review]
:shipit:

**Review by Benehiko — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**